Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -325,28 +325,35 @@ class BrowserViewController: UIViewController {
 
             addChildViewController(homePanelController!)
         }
+
+        // We have to run this animation, even if the view is already showing because there may be a hide animation running
+        // and we want to be sure to override its results.
+        UIView.animateWithDuration(0.2, delay: 0, options: .BeginFromCurrentState, animations: { () -> Void in
+            self.homePanelController!.view.alpha = 1
+        }, completion: { finished in
+            if finished {
+                self.webViewContainer.accessibilityElementsHidden = true
+                UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, nil)
+            }
+        })
         toolbar?.hidden = !inline
         view.setNeedsUpdateConstraints()
     }
 
     private func hideHomePanelController() {
         if let controller = homePanelController {
-            UIView.animateWithDuration(0.2, animations: { () -> Void in
-                    controller.view.alpha = 0
-                }, completion: { _ in
+            UIView.animateWithDuration(0.2, delay: 0, options: .BeginFromCurrentState, animations: { () -> Void in
+                controller.view.alpha = 0
+            }, completion: { finished in
+                if finished {
                     controller.view.removeFromSuperview()
                     controller.removeFromParentViewController()
                     self.homePanelController = nil
                     self.webViewContainer.accessibilityElementsHidden = false
                     self.toolbar?.hidden = false
-                    self.startTrackingAccessibilityStatus()
```

Comment:
We just added this yesterday -- are you sure it's safe to remove? @dusek, can you give a quick summary of why you're calling this here?